src/ralph/back_office/helpers.py

```diff
@@ -10,3 +15,36 @@ def get_email_context_for_transition(transition_name: str) -> EmailContext:
         'body': 'Please see documents provided in attachments for "{}".'.format(transition_name)  # noqa
     }
     return EmailContext(**default)
+
+
+def status_converter(
+    old_status_id, target_status_id, from_status_cls=BackOfficeAssetStatus,
+    to_status_cls=DataCenterAssetStatus
+):
+    """
+    Convert BackOfficeAsset (BO) status to DatacenterAsset (DC) status (or the
+    other way round).
+    Try using status with the same name.
+    If no status with such name exists, use name of the default status from
+    transition.
+    If no status with such name exists, use default status from settings.
+
+    Args:
+        old_status_id: BO status (int)
+        target_status_id: ID of default status assigned to transition
+        from_status_cls: Source status class
+        to_status_cls: Destination status class
+
+    Returns:
+        new_status_id: DC status (int)
+    """
+    try:
+        if not target_status_id:
```

Comment:
IMO it's better to move all code that can't cause exception outside of `try` block.